canonical/lxd-ui
# Improve error notification for instance bulk actions [WD-3867]

## Done

- Improved error notification for instance bulk actions by including a summary of which instance actions succeeded and which failed

Fixes [WD-3867](https://warthogs.atlassian.net/browse/WD-3867)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Go to instance list
    - Trigger a start bulk action on some instances that are expected to fail (e.g. assign the instance a profile with no root storage device) as well as some that should succeed
    - Check the "partial failure" error notification
